manifests/cluster-network-addons/0.4.0/cluster-network-addons-operator.0.4.0.clusterserviceversion.yaml

```diff
@@ -151,6 +151,8 @@ spec:
                     value: docker.io/nfvpe/multus:latest
                   - name: LINUX_BRIDGE_IMAGE
                     value: quay.io/kubevirt/cni-default-plugins:latest
+                  - name: LINUX_BRIDGE_MARKER_IMAGE
```

Comment:
we should not change any manifest file.
After this PR is merge we can open a new one with a new release

Reply:
removed